Source: Blender upstream
Upstream: yes, part of Blender SVN revision 44635
Reason: Fix compilation of OpenJPEG

--- trunk/blender/source/blender/imbuf/intern/jp2.c	2012/02/27 10:35:39	44479
+++ trunk/blender/source/blender/imbuf/intern/jp2.c	2012/03/04 11:58:55	44635
@@ -329,7 +329,7 @@
 
 static void cinema_parameters(opj_cparameters_t *parameters)
 {
-	parameters->tile_size_on = false;
+	parameters->tile_size_on = FALSE;
 	parameters->cp_tdx=1;
 	parameters->cp_tdy=1;
 	
